data/sites.json

```diff
@@ -305,5 +305,9 @@
             { "street": "166 Walnut St", "city": "Saugus", "zip": "01906" },
             { "street": "296 Buffinton St", "city": "Somerset", "zip": "02726" }
         ]
+    },
+    "BaystateHealth": {
+        "name": "Baystate Health",
+        "website": "https://www.baystatehealth.org/covid19"
```

Comment:
I think you want this to be signUpLink - we should always return a signUpLink that will be the website users are kicked out to when appointments are available and they try to book.